metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_utils.py

```diff
@@ -59,6 +62,44 @@ class SnowflakeCommonMixin:
 
     platform = "snowflake"
 
+    @staticmethod
+    @lru_cache(maxsize=128)
+    def create_snowsight_base_url(account_id: str) -> Optional[str]:
+        cloud: Optional[str] = None
+        account_locator: Optional[str] = None
+        cloud_region_id: Optional[str] = None
+
+        if "." not in account_id:  # e.g. xy12345
+            account_locator = account_id.lower()
+            cloud_region_id = "us-west-2"
```

Comment:
This is the default? Maybe extract this into a constants

DEFAULT_REGION_ID

at the top?